sign/signup.js

```diff
@@ -0,0 +1,98 @@
+import {
+  emailCheck,
+  errorColorChange,
+  errorColorRomove
+} from "./utils.js";
+
+
+//이메일 input에서 focus out 할 때, 이메일 형식에 맞지 않는 값이 있는 경우 아래에 “올바른 이메일 주소가 아닙니다.” 에러 메세지를 보입니다.
+// 로그인 버튼 클릭 또는 Enter키 입력으로 로그인 실행돼야 합니다
+//이메일 input에서 focus out 할 때, 값이 없을 경우 아래에 “이메일을 입력해 주세요.” 에러 메세지를 보입니다.
+//이메일 input에서 focus out 할 때, input 값이 test@codeit.com 일 경우, “이미 사용 중인 이메일입니다.” 에러 메세지를 보입니다.
+const email = document.querySelector('#email');  //input태그
+const emailError =  document.querySelector('#email-error-message'); //이메일p 태그
+
+function emailErrorMessageHandler(e){
+  if(!email.value){
+    emailError.textContent =  '이메일을 입력해 주세요.';
+    errorColorChange(email);
+  }else if(!emailCheck(email.value)){
+    emailError.textContent = '올바른 이메일 주소가 아닙니다.';
+    errorColorChange(email);
+  }else if(email.value === "test@codeit.com"){
+    emailError.textContent = '이미 사용 중인 이메일입니다.';
+    errorColorChange(email);
+  }else{
+    emailError.textContent = '';
+    errorColorRomove(email);
+  }
+}
+email.addEventListener("focusout",emailErrorMessageHandler);
+
+
+
+
+//비밀번호 input에서 focus out 할 때, 값이 8자 미만으로 있거나 문자열만 있거나 숫자만 있는 경우, 
+//“비밀번호는 영문, 숫자 조합 8자 이상 입력해 주세요.” 에러 메세지를 보입니다.
+// 비밀번호 input에서 focus out 할 때, 값이 없을 경우 아래에 “비밀번호를 입력해 주세요.” 에러 메세지를 보입니다.
+const password = document.querySelector('#password');  //input태그
+const PasswordError =  document.querySelector('#password-error-message'); //이메일p 태그
```

Comment:
일관성있게 네이밍을 카멜케이스로 변경하면 좋을것 같아요!
```suggestion
const passwordError =  document.querySelector('#password-error-message'); //이메일p 태그
```